dbschema/scripture.esdl

```diff
@@ -7,6 +7,7 @@ module Scripture {
     multi verses: VerseRange {
       readonly := true;
       on source delete delete target;
+      on target delete deferred restrict;
```

Comment:
Similar question here...why exactly is this line needed?

Reply:
I believe when we say `delete Object` it's compiled to some SQL like
```sql
delete from Users;
delete from Scripture::VerseRange;
delete from Scripture::Collection;
```
Since abstract queries are compiled to a union of their concretes.
And in this case `VerseRange` is enumerated before `Collection`, so an error is thrown.
The `deferred restrict` means the VerseRange can be deleted, as long as the `Collection` is also deleted within the same transaction.